docs/syncserver/README.md

```diff
@@ -10,13 +10,27 @@ the build products and runtime dependencies from the rest of your system.
 
 - [x] [Docker](https://docs.docker.com/get-started/)
 
+| **Aspect**               | **Dockerfile**                                             | **Dockerfile.distroless**                                 |
+| ------------------------ | ---------------------------------------------------------- | --------------------------------------------------------- |
+| **Shell & Tools**        | ✅ Includes shell and tools                                | ❌ Minimal, no shell or tools                             |
+| **Debugging**            | ✅ Easier debugging with shell and tools                   | ❌ Harder to debug due to minimal environment             |
+| **Health Checks**        | ✅ Supports complex health checks                          | ❌ Health checks need to be simple or directly executable |
+| **Image Size**           | ❌ Larger image size                                       | ✅ Smaller image size                                     |
+| **Customization**        | ✅ Easier to customize with additional packages            | ❌ Limited customization options                          |
+| **Attack Surface**       | ❌ Larger attack surface due to more installed packages    | ✅ Reduced attack surface                                 |
+| **Libraries**            | ✅ More libraries available                                | ❌ Limited libraries                                      |
+| **Start-up Time**        | ❌ Slower start-up time due to larger image size           | ✅ Faster start-up time                                   |
+| **Tool Compatibility**   | ✅ Compatible with more tools and libraries                | ❌ Compatibility limitations with certain tools           |
+| **Maintenance**          | ❌ Higher maintenance due to larger image and dependencies | ✅ Lower maintenance with minimal base image              |
+| **Architecture Support** | amd64, arm64                                               | amd64, arm64, arm, s390x, ppc64le                         |
```

Comment:
This table is a great addition for people who aren't sure which one to pick. My only question is: have you tested that arm, s390x and ppc64le work? I would be somewhat surprised if all our dependencies work with them.

Reply:
Apologies for missing this earlier! I was experimenting with multi-architecture builds over the weekend and encountered some unexpected issues, especially with the `arm` architecture. While the build is working fine for `amd64` and ` arm64` I ran into the following error when attempting to build:

```bash
Copy code
4.397 error: failed to get `percent-encoding-iri` as a dependency of package `anki v0.0.0 (/app/rslib)`
4.397 
4.397 Caused by:
4.398   failed to load source for dependency `percent-encoding-iri`
```
will remove this line for now.